test/framework/src/main/java/org/elasticsearch/test/rest/client/RestTestClient.java

```diff
@@ -69,35 +52,29 @@
  * Wraps a {@link RestClient} instance used to send the REST requests.
  * Holds the {@link RestSpec} used to translate api calls into REST calls
  */
-public class RestTestClient implements Closeable {
-
-    public static final String PROTOCOL = "protocol";
-    public static final String TRUSTSTORE_PATH = "truststore.path";
-    public static final String TRUSTSTORE_PASSWORD = "truststore.password";
-
+public class RestTestClient {
```

Comment:
shall we consider renaming this class given that we want to move away from rest tests naming? Also RestTestResponse? maybe in a followup? WDYT?